spec/statements.md

```diff
@@ -40,45 +51,53 @@ If a statement can possibly be reached by execution, the statement is said to be
 
 \[*Example*: In the following code
 
-void F() {\
-Console.WriteLine("reachable");\
-goto Label;\
-Console.WriteLine("unreachable");\
-Label:\
-Console.WriteLine("reachable");\
+```csharp
+void F() {
+Console.WriteLine("reachable");
+goto Label;
+Console.WriteLine("unreachable");
+Label:
+Console.WriteLine("reachable");
 }
+```
 
 the second invocation of Console.WriteLine is unreachable because there is no possibility that the statement will be executed. *end example*\]
 
 A warning is reported if the compiler determines that a statement is unreachable. It is specifically not an error for a statement to be unreachable.
 
-\[*Note*: To determine whether a particular statement or end point is reachable, the compiler performs flow analysis according to the reachability rules defined for each statement. The flow analysis takes into account the values of constant expressions (§12.20) that control the behavior of statements, but the possible values of non-constant expressions are not considered. In other words, for purposes of control flow analysis, a non-constant expression of a given type is considered to have any possible value of that type.
+> [!NOTE]
+> To determine whether a particular statement or end point is reachable, the compiler performs flow analysis according to the reachability rules defined for each statement. The flow analysis takes into account the values of constant expressions (§12.20) that control the behavior of statements, but the possible values of non-constant expressions are not considered. In other words, for purposes of control flow analysis, a non-constant expression of a given type is considered to have any possible value of that type.
 
 In the example
 
-void F() {\
-const int i = 1;\
-if (i == 2) Console.WriteLine("unreachable");\
+```csharp
+void F() {
+const int i = 1;
+if (i == 2) Console.WriteLine("unreachable");
 }
+```
 
 the Boolean expression of the if statement is a constant expression because both operands of the == operator are constants. As the constant expression is evaluated at compile-time, producing the value false, the Console.WriteLine invocation is considered unreachable. However, if i is changed to be a local variable
 
-void F() {\
-int i = 1;\
-if (i == 2) Console.WriteLine("reachable");\
+```csharp
+void F() {
+int i = 1;
+if (i == 2) Console.WriteLine("reachable");
```

Comment:
not certain if lack of indentation is intended in tehse examples.

Reply:
It's not. that's an artifact of the conversion process, and should be addressed.